community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPageCursor.java

```diff
@@ -776,4 +776,20 @@ public void setCursorException( String message )
             this.cursorException = message;
         }
     }
+
+    @Override
+    public void clear()
```

Comment:
I'd prefer to call it `zapPage` or something like that, to avoid confusing it with `ByteBuffer.clear()`.